skietbaan_fe/src/components/Login.js

```diff
@@ -95,65 +105,73 @@ class Login extends Component {
     }
   }
 
+  togglePassword() {
+    var x = document.getElementById("PasswordValue");
+    if (x.type === "password") {
+      x.type = "text";
+    } else {
+      x.type = "password";
+    }
+  }
+
   render() {
+    document.addEventListener('DOMContentLoaded', () => {
+      this.ToggleNavbar();
+   }, false);
     if(getCookie("token")){
       window.location = "/home";
     }
-    let invalidPasswordMessage;
-    let invalidUsernameMessage;
-
-    if (this.state.invalidPassword) {
-      invalidPasswordMessage = <div className="invalid-message">Please choose a password</div>;
-    }
-    if (this.state.invalidUsername) {
-      invalidUsernameMessage = <div className="invalid-message">Please enter your username</div>;
-    }
 
     return (
-      <Container className="App">
+      <div className="Page-content">
+      <div className = "welcome-header"><label className="welcome-label">Welcome back to skietbaan</label></div>
       <div className="header-container">
-      <h2>Login</h2>
+      <label className = "header-label">Login</label>
       </div>
         <div className="centre-login">
           <Form className="form" autoComplete="off">
 
             <Col className="no-padding">
               <FormGroup>
-                <Label className="front-white"> Type <strong>Username</strong></Label>
-                <Input
+                <label className="front-white input-label">Username <div 
+                className={this.state.invalidUsername ? "invalid-icon" :"hidden"}></div></label>
+                
+                <div className="input-container">
+                <input
                   type="text"
                   name="usernameValue"
                   id="us"
-                  placeholder="username"
                   value={this.state.usernameValue}
                   onChange={this.handleChange}
-                  className={this.state.invalidUsername ? "invalid" : ""}
+                  className= "input"
                 />
-                {invalidUsernameMessage}
+                </div>
               </FormGroup>
             </Col>
             <Col className="no-padding">
               <FormGroup>
-                <Label className="front-white" for="examplePassword">Type <strong>Password</strong></Label>
-                <Input
+                <label className="front-white input-label" for="examplePassword">
+                Password <div className={this.state.invalidPassword ? "invalid-icon":"hidden"}></div></label>
+                <div className="input-container">
+                <input
                   type="password"
                   name="passwordValue"
-                  id="examplePassword"
-                  placeholder="********"
+                  id="PasswordValue"
```

Comment:
Keep consistency with your naming.
PasswordValue is not in camelCase notation

Reply:
Fixed